std/zip.d

```diff
@@ -1,7 +1,7 @@
 // Written in the D programming language.
 
 /**
- * Read/write data in the $(LINK2 http://www.info-_zip.org, zip archive) format.
+ * Read/write data in the $(LINK2 http://www.info-zip.org, zip archive) format.
```

Comment:
I think you have to put an underscore before the second "zip", here. Otherwise it generates an `<a>` element for an anchor, and that messes with the `<a>` from the `LINK2`.